config/commentions.php

```diff
@@ -18,4 +18,10 @@
      */
     'allow_edits' => true,
     'allow_deletes' => true,
+
+    /**
+     * Allowed reactions for comments.
+     * Define the list of emojis that users can react with.
+     */
+    'allowed_reactions' => ['👍', '❤️', '😂', '😮', '😢', '🤔'],
```

Comment:
Based on Erik's feedback, maybe this will change, but I do wonder if this stays, if we should include a way to allow all emojis?

Reply:
Yeah, I guess that's fair if we want to do it as follow-up. I wasn't sure how complicated it would be.